src/main/java/gmap_distance_matrix/Main.java

```diff
@@ -1,7 +1,140 @@
 package gmap_distance_matrix;
 
```

Comment:
Since everything is in the Main, it is most probably violating SRP because one class is creating requests, handling responses and also doing I/O operations. This means Main class has more than one reason to change.